tests/webview/ui/test_content.py

```diff
@@ -766,3 +766,18 @@ def test_chapter_review_version_matches_book_version(webview_base_url, selenium,
 
     # THEN the chapter review version matches the book version
     assert content.page_version == content.book_version
+
+
+@markers.webview
+@markers.test_case('C195071')
+@markers.nondestructive
+@markers.parametrize('id', ['yWshIYVW@1'])
+def test_buggy_msg_not_show_in_left_panel(webview_base_url, id, selenium):
```

Comment:
I realize the name of the test included buggy load message but we also want to take a second to reflect on exactly what we are testing here. Remember: we are about quality in all things ;). As I read the test case what is REALLY being tested is that when the `This page is in books` message exists in the left pane that it has books listed underneath it.

Please rename the test to reflect what your are testing. A better name may be:
`test_this_page_in_books_contains_books`